src/components/Button.js

```diff
@@ -0,0 +1,71 @@
+import React from 'react'
+import PropTypes from 'prop-types'
+import Text from './Text'
+import { navigate as NavTrial } from 'gatsby'
+
+const StyledLink = ({
+  children,
+  border_color,
+  connect,
+  background,
+  margin_left,
+  buttonWidth,
+  mobileSize,
+}) => (
+  <>
+    <button
+      onClick={() => {
+        NavTrial(connect)
+      }}
+    >
+      {children}
+    </button>
+    <style jsx>{`
+      button {
+        vertical-align: middle;
+        padding: 0.5rem 2rem;
+        ${background ? `background-color: ${background};` : ''}
+        border-radius: 350px;
+
+        cursor: pointer;
+        max-width: 200ch;
+        max-height: 100%;
+        text-align: center;
+        border-style: solid;
+        color: white;
+
+        ${border_color ? `border-color: ${border_color};` : ''}
+        font-weight:700;
+
+        ${margin_left ? `margin-left: ${margin_left};` : ''}
+        font-size:1.25rem;
+        font-family: KeepCalm;
+        font-weight: bold;
+      }
+
+      button:hover {
+        background-color: #48e9ce;
```

Comment:
Are there variables for these?

Reply:
I don't think they currently exist in the codebase, but they could be added to `theme.js`, something like
```
const c_NEON_TEAL = '#48e9ce'
```